crates/sui-source-validation-service/src/lib.rs

```diff
@@ -361,7 +361,10 @@ pub async fn verify_packages(config: &Config, dir: &Path) -> anyhow::Result<Netw
     Ok(lookup)
 }
 
-pub async fn watch_for_upgrades(config: &Config) -> anyhow::Result<()> {
+pub async fn watch_for_upgrades(
+    config: &Config,
+    _app_state: Arc<RwLock<AppState>>,
+) -> anyhow::Result<()> {
```

Comment:
Main objective: `watch_for_upgrades` has a handle to `_app_state`. It is unused right now, see follow up PRs for more.